src/platform/datapath_winuser.c

```diff
@@ -552,6 +718,72 @@ CxPlatSocketContextUninitializeComplete(
     _In_ CXPLAT_SOCKET_PROC* SocketProc
     );
 
+void*
+RioRecvBufferAllocate(
```

Comment:
Might be nice to redefine the definition of the pool alloc/free functions do you can do something like:
```c
CXPLAT_POOL_ALLOC RioRecvBufferAllocate;
CXPLAT_POOL_FREE RioRecvBufferFree;
```

Reply:
Yeah, I can do this if you want, or we can save that for a follow-up PR. (Do we want to keep this PR scoped to datapath_winuser.c?)